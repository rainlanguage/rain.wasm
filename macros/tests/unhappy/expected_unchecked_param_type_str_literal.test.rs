#[macro_use]
extern crate wasm_bindgen_utils_macros;

struct TestStruct;

#[wasm_export]
impl TestStruct {
    #[wasm_export]
    pub async fn test_method(
        #[wasm_export(unchecked_param_type = notStringLiteral)]
        arg: wasm_bindgen::JsValue
    ) -> Result<String, Error> {
        Ok("test".to_string())
    }
}

fn main() {}