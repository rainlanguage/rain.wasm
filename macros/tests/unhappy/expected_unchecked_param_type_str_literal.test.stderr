error: expected string literal
  --> tests/unhappy/expected_unchecked_param_type_str_literal.test.rs:10:23
   |
10 |         #[wasm_export(unchecked_param_type = notStringLiteral)]
   |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
